apps/coscrad-frontend/src/components/resources/terms/term-detail.presenter.css

```diff
@@ -31,7 +31,7 @@
 }
 
 #media-player {
-    margin: auto !important;
+    margin: 0 auto !important;
```

Comment:
Are you proposing that we use **both** css modules **and** `styled-components` ?

Reply:
I'm guessing you're just trying out certain things with style-components and leaving other CSS in place.  We'd fully convert to styled components in future though